modules/cluster-logging-loki-tech-preview.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-loki-tech-preview_{context}"]
+:FeatureName: Loki Operator
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-loki"]
+= About Loki
+
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system currently offered as an alternative to Elasticsearch as a log store for the {logging}.
+
+== Deploying the Lokistack
+You can use the {product-title} web console to install the LokiOperator.
+
+To install the LokiOperator using the {product-title} web console:
+
+. Install the LokiOperator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *LokiOperator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces on the cluster*.
+
+.. Under *Installed Namespace*, select *openshift-operators-redhat*.
++
+You must specify the `openshift-operators-redhat` namespace. The `openshift-operators`
+namespace might contain Community Operators, which are untrusted and could publish
+a metric with the same name as an {product-title} metric, which would cause
+conflicts.
+
+.. Select *Enable operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+You must select this option to ensure that cluster monitoring
+scrapes the `openshift-operators-redhat` namespace.
+
+.. Select an *Approval Strategy*.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+.. Verify that you installed the LokiOperator. Visit the *Operators* → *Installed Operators* page and look for "LokiOperator."
+
+.. Ensure that *LokiOperator* is listed in all the projects whose *Status* is *Succeeded*.
+
+[NOTE]
+====
+You do not have to specify a secret in the `ClusterLogForwarder` CR if Vector is forwarding to Loki using token authentication.
+====
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://grafana.com/docs/loki/latest/[Loki Documentation]
```

Comment:
The Loki Documentation doesn't include anything about how to deploy lokistack via Operator.   it is better to link the PR If had created.  or else https://github.com/grafana/loki/tree/main/operator/docs

Reply:
Good catch @anpingli (sorry for being late on reviewing the docs). But I suggest to add some simple references as:
1. To install a simple lokistack use [manifest](https://github.com/grafana/loki/blob/main/operator/hack/lokistack_gateway_ocp.yaml)
2. To forward logs from fluentd/vector on OpenShift link: https://github.com/grafana/loki/blob/main/operator/docs/forwarding_logs_to_gateway.md#openshift-logging

For 1. above we need some docs to:
-  Explain users how to create and reference on the object storage secret (e.g. for [AWS S3](https://github.com/grafana/loki/blob/main/operator/hack/deploy-aws-storage-secret.sh))
- As well a small table explaining the CPU/Mem and PVC requirements when selecting one of the three sizes `1x.extra-small`, `1x.small`, 1x.medium` as below:

| Size | Total CPU required | Total Mem required | Total PVC required | Description |
| ------ | ------------------------: | -------------------------: | ------------------------: | --------------- |
| 1x.extra-small | TBD | TBD | TBD | This size provides an non-HA Loki installation for demo-purposed only. DO NOT USE IN PRODUCTION |
| 1x.small |  TBD | TBD | TBD | This size provide an HA Loki installation with single replication factor. Means all chunks store operations are replicated to two ingesters at a time. The Log chunks are stored once on object storage, means only the succeeding ingester wins the race. It targets Log Volume ingestion of 250GB/day and Query Traffic of 50 req/s |
| 1x.small |  TBD | TBD | TBD | This size provide an HA Loki installation with double replication factor. Means all chunks store operations are replicated to three ingesters at a time. The Log chunks are stored once on object storage, means only the succeeding ingester wins the race. It targets Log Volume ingestion of 2TB/day and Query Trafic of 75 req/s |

I will post here the TBD for CPU/Mem/PVC later.

Reply:
> The Loki Documentation doesn't include anything about deploy lokistack via Operator. it is better to link the PR If had created or else link these https://github.com/grafana/loki/tree/main/operator/docs

Was more included for general Loki additional info -- moved up to the bottom of the about section.

Reply:
@periklis / @anpingli - Additional docs for Loki are covered by RHDEVDOCS-3138, we wouldn't include everything about Loki in the Release Notes... just, this exists, here's how to turn it on.

Reply:
Fine for me if we are tracking this in a follow up ticket